content/version/1/4/code-of-conduct.de.md

```diff
@@ -14,16 +14,16 @@ Im Interesse der Förderung eines offenen und einladenden Umfeldes wollen wir un
 Beispiele für Verhaltensweisen, welche dazu beitragen ein positives Umfeld zu erzeugen, beinhalten:
 
 * Die Verwendung von einladenden und einbindenden Formulierungen
-* Abweichenden Ansichten und Erfahrungen wird Respekt entgegengebracht
-* Konstruktive Kritik wird höflich entgegengenommen
-* Fokussierung auf das, was das Beste ist für die Gemeinschaft
-* Verständnis zeigen gegenüber anderen Mitgliedern der Gemeinschaft
+* Respektieren von abweichenden Ansichten und Erfahrungen
+* Höfliches Entgegennehmen konstruktiver Kritik
+* Fokussierung auf das Beste für die Gemeinschaft
```

Comment:
Maybe something like "Fokussierung auf das Wohl der Gemeinschaft" ?
While the above is a very literal translation it kind of seems...off to my ears